src/Audio/Effects/ReverbEffect.cs

```diff
@@ -23,7 +22,8 @@ public ReverbEffect(int source)
             ALC.EFX.Effect(effect, EffectFloat.ReverbGain, 0.23f);
 
             ALC.EFX.AuxiliaryEffectSlot(slot, EffectSlotInteger.Effect, effect);
-            ALC.EFX.Source(source, EFXSourceInteger3.AuxiliarySendFilter, slot, 0, 0);
+
+            return (int)EffectType.Reverb;
```

Comment:
`SoundEffect.effect` is an internal ID of the OpenAL effect
```suggestion
            return effect;
```

Reply:
Done 👍